src/main/java/com/exasol/adapter/jdbc/RemoteConnectionException.java

```diff
@@ -15,4 +15,13 @@
     public RemoteConnectionException(final String message, final Throwable cause) {
         super(message, cause);
     }
+
```

Comment:
If you update a serializable class, please regenerate the UID!

Reply:
done